test/loco/integer/sendmoremoney3.clj

```diff
@@ -0,0 +1,109 @@
+(ns ^{:author "Andrea Richiardi"
+      :doc "Send + More = Money solution #3 + tests. This solution differs from #2 as it models each
+            individual letter as decision variable and then constaints the common ones
+            explicitely. It does not make use of loco's $scalar constraint (see sendmoremoney.clj
+            for a more concise implementation)." }
+  loco.integer.sendmoremoney3
```

Comment:
Usually I see the value of including multiple styles of solutions, but I really don't see why it makes sense to include this one. It seems pretty clear to me that this is strictly less efficient than the other two solutions, as there are just more variables that make it harder for the engine to propagate the constraints. I think it makes more sense to just remove this namespace, unless you can point me to a definitive source where this model came from.